util/SerializeEmpire.cpp

```diff
@@ -186,7 +186,7 @@ void Empire::serialize(Archive& ar, const unsigned int version)
             & BOOST_SERIALIZATION_NVP(m_research_progress)
             & BOOST_SERIALIZATION_NVP(m_production_queue)
             & BOOST_SERIALIZATION_NVP(m_available_building_types)
-            & BOOST_SERIALIZATION_NVP(m_available_part_types)
+            & BOOST_SERIALIZATION_NVP(m_available_ship_parts)
```

Comment:
Breaks save loading: `LoadGame(...) failed!  Error: XML start/end tag mismatch - m_available_ship_parts`. Need to increment the serialization class version and have a case to handle the older field name using `make_nvp`.

Reply:
It's probably better to use `make_nvp` and preserve the tag instead of changing the format.

Reply:
Sounds reasonable for just renaming without changing the data being encoded.